app/api/tax.py

```diff
@@ -43,9 +65,38 @@ class Meta:
                          type_='event')
 
 
-class TaxList(ResourceList):
+class TaxListPost(ResourceList):
     def before_post(self, args, kwargs, data):
         require_relationship(['event'], data)
+        if not has_access('is_coorganizer', event_id=data['event']):
+            raise ForbiddenException({'source': ''}, 'Co-organizer access is required.')
+
+    def before_get(self, args, kwargs):
+        if has_access('is_admin'):
```

Comment:
Actually I was saying that current "is_admin" will throw error if the user is not logged in but I think here guests should be able to view the public schema but they would not be able to since "is_admin" has already raised the permission error. I guess this is the reason behind "is_coorganizer_Wthout_jwt.." also.